sqlite/ext/comdb2/systables.c

```diff
@@ -0,0 +1,175 @@
+/*
```

Comment:
I prefer ezsystable (`create_system_table()`) interface because it makes creating a new systable a lot simpler:

https://github.com/bloomberg/comdb2/blob/master/sqlite/ext/comdb2/cluster.c

Reply:
Can you iterate the hashtable and store the elements in the array in `create_system_table()`'s init callback (the 3rd argument)?

Reply:
For instance, `bdb_fill_cluster_info()` converts netinfo to an array which eztable expects.

Reply:
Okay then.